nbs/introduction.ipynb

```diff
@@ -89,147 +89,147 @@
      "name": "stderr",
```

Comment:
<p>why is this error message here?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/DeanLight/spanner_workbench/pull/134/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='91'/>